pkg/kubelet/kubelet.go

```diff
@@ -2291,7 +2291,7 @@ func getStreamingConfig(kubeCfg *kubeletconfiginternal.KubeletConfiguration, kub
 		SupportedPortForwardProtocols:   streaming.DefaultConfig.SupportedPortForwardProtocols,
 	}
 	if !crOptions.RedirectContainerStreaming {
-		config.Addr = net.JoinHostPort("localhost", "0")
+		config.Addr = ""
```

Comment:
From your comment, I'm understanding that setting to `""` means that now we are listening on more IP address than just localhost? Is this a security concern?

Is there an option where we still specify `localhost` but do so in a way that doesn't require ipv6?

Regardless of the path forward, it'd be great to have a descriptive variable/function name to make it clearer the purpose of `config.Addr = ""`.

Reply:
> From your comment, I'm understanding that setting to "" means that now we are listening on more IP address than just localhost? Is this a security concern?

that's my concern too, I really don't know what's the purpose of using `localhost`, that`s why assigned the PR to the patch author :)

> Is there an option where we still specify localhost but do so in a way that doesn't require ipv6?

The problem is, AFAIK,  this depends on the OS mechanism to resolve `localhost` and its preference. 

> Regardless of the path forward, it'd be great to have a descriptive variable/function name to make it clearer the purpose of config.Addr = "".

Is not the address where the streaming server is listening?

Reply:
The problem seems to be that "localhost" is resolving to an IPv6 address, but IPv6 is disabled, right?

Isn't that just a bad configuration on the machine?